lib/info-extras.js

```diff
@@ -268,3 +268,54 @@ exports.cleanVideoDetails = videoDetails => {
     'videoDetails.shortDescription', 'videoDetails.description');
   return videoDetails;
 };
+
+/**
+ * Get storyboards info.
+ *
+ * @param {Object} info
+ * @returns {Array.<Object>}
+ */
+exports.getStoryboards = info => {
+  const parts = info.player_response.storyboards &&
+    info.player_response.storyboards.playerStoryboardSpecRenderer &&
+    info.player_response.storyboards.playerStoryboardSpecRenderer.spec &&
+    info.player_response.storyboards.playerStoryboardSpecRenderer.spec.split('|');
+
+  if (!parts) return [];
+
+  const url = new URL(parts.shift());
+
+  return parts.map((part, i) => {
+    let [
+      width,
+      height,
+      count,
+      storyboardWidth,
+      storyboardHeight,
+      interval,
+      nameReplacement,
```

Comment:
name replacement can be "M$M", but when that is added to the url, the url gives a 404. i replaced it with "M0" (by looking at the network tab) and that worked

Reply:
hints the name `templateUrl`, it's meant to edited before request, changing `$M` to the index of the story board based on `storyboardCount`. And BTW these fields are provided by YouTube, not me 😉

Reply:
Adding an example would clearify the use of storyboards

Reply:
ah i see. yep that wasn't too clear to me when i was looking at it